apps/web/src/routes/AuthLayoutRouter/components/ContextMenu/ContextMenu.jsx

```diff
@@ -533,53 +533,53 @@ function ListItemRenderer ({ item, rootPath, groupSlug, canDnd, isOverlay = fals
+                externalLink={item?.customView?.type === 'externalLink' ? item.customView.externalLink : null}
+                className='text-base text-foreground border-2 border-foreground/20 hover:border-foreground/100 hover:text-foreground rounded-md p-2 bg-background text-foreground mb-[.5rem] w-full transition-all scale-100 hover:scale-105 opacity-85 hover:opacity-100 flex items-center justify-between'
               >
                 <div>
                   <WidgetIconResolver widget={item} />
-                  <span className="text-base ml-2">{itemTitle}</span>
+                  <span className='text-base ml-2'>{itemTitle}</span>
                 </div>
                 {isItemDraggable && <GrabMe {...itemListeners} {...itemAttributes} />}
               </MenuLink>
-            );
-          } else if (rootPath !== "/my" && rootPath !== "/all" && !item.title) {
+            )
+          } else if (rootPath !== '/my' && rootPath !== '/all' && !item.title) {
             return (
               <MenuLink
                 to={itemUrl}
-                externalLink={item?.customView?.type === "externalLink" ? item.customView.externalLink : null}
-                className="transition-all px-2 pb-2 text-foreground scale-1 hover:scale-110 scale-100 hover:text-foreground opacity-80 hover:opacity-100 flex align-items justify-between"
+                externalLink={item?.customView?.type === 'externalLink' ? item.customView.externalLink : null}
+                className='transition-all px-2 pb-2 text-foreground scale-1 hover:scale-110 scale-100 hover:text-foreground opacity-80 hover:opacity-100 flex align-items justify-between'
               >
                 <div>
                   <WidgetIconResolver widget={item} />
-                  <span className="text-base ml-2">{itemTitle}</span>
+                  <span className='text-base ml-2'>{itemTitle}</span>
                 </div>
                 {isItemDraggable && <GrabMe {...itemListeners} {...itemAttributes} />}
               </MenuLink>
-            );
-          } else if (rootPath === "/my" || rootPath === "/all" || rootPath !== "/members" || (item.title && item.type !== "chat")) {
+            )
+          } else if (rootPath === '/my' || rootPath === '/all' || rootPath !== '/members' || (item.title && item.type !== 'chat')) {
             return (
               <MenuLink
                 to={itemUrl}
-                externalLink={item?.customView?.type === "externalLink" ? item.customView.externalLink : null}
-                className="text-base text-foreground border-2 border-foreground/20 hover:border-foreground/100 hover:text-foreground rounded-md p-2 bg-background text-foreground mb-[.5rem] w-full transition-all scale-100 hover:scale-105 opacity-85 hover:opacity-100 flex align-items justify-between"
+                externalLink={item?.customView?.type === 'externalLink' ? item.customView.externalLink : null}
```

Comment:
In general, I plan to go over all of this and double-check if things can be refractored from aarons changes

Reply:
yeah, I was going to go over better use of MenuLink with him at some point too